### PR TITLE
Terms and condition

### PR DESCRIPTION
**Pull request by roylin2000**
# Description

Finished polishing up the terms and conditions and privacy policy page.

Fixes # (issue/ticket number)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (if required)
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: tpp-app/src/home/pages/PrivacyPolicyScreen.js [hunk 1]
```diff
@@ -1,14 +1,26 @@
...
-import { View, ScrollView, StyleSheet, Text, useWindowDimensions, TouchableOpacity, ImageBackground, SafeAreaView} from 'react-native';
-import RenderHtml from 'react-native-render-html';
+import { View, ScrollView, StyleSheet, Text, useWindowDimensions, TouchableOpacity, ImageBackground, Linking} from 'react-native';
 import BackIcon from '../../../ios/tppapp/Images.xcassets/icons/back_icon.svg'
 import OnboardingBackground from '../../../ios/tppapp/Images.xcassets/SplashScreenBackground.imageset/colourwatercolour.png'
 
+const EmailLink = () => {
+    const openLink = () => {
+        
+            Linking.canOpenURL("mailto:hello@periodpurse.com").then(() => {
```
Comment (annie951977) [inline tpp-app/src/home/pages/PrivacyPolicyScreen.js:9]: The mail to link doesn't seem to work

**Review by annie951977 — CHANGES_REQUESTED** [review]
image
I'm getting this error when I try to click the link!

File: tpp-app/src/home/pages/PrivacyPolicyScreen.js (hunk 1 above)
Comment (helen-li) [reply tpp-app/src/home/pages/PrivacyPolicyScreen.js:9]: i get this as well but roy you tested on faye's phone and it works alright there?

**Review by helen-li — APPROVED** [review]
just annie's comment otherwise nicee

File: tpp-app/src/home/pages/PrivacyPolicyScreen.js (hunk 1 above)
Comment (roylin2000) [reply tpp-app/src/home/pages/PrivacyPolicyScreen.js:9]: works on phone :)

**Review by annie951977 — APPROVED** [review]
Noice

**Pull request merged**